.vib/magento/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,57 @@
+const CLICK_DELAY = 0;
+const GET_DELAY = 500;
+const BASE_URL = 'http://vmware-magento.my';
+
+for (const command of ['click']) {
+  Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+    const origVal = originalFn(...args);
+
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(origVal);
+      }, CLICK_DELAY);
+    });
+  });
+}
+
+for (const command of ['get']) {
+  Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+    const origVal = originalFn(...args);
+
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(origVal);
+      }, GET_DELAY);
+    });
+  });
+}
+
+// Due to a bug when using "hosts" in Cypress, we cannot set a "baseUrl" in the
+// cypress.json file. Workaround this by modifying the "visit" command to preprend
+// the base URL.
+//
+// Further details: https://github.com/cypress-io/cypress/issues/20647
+Cypress.Commands.overwrite('visit', (originalFn, url, options) => {
+  return originalFn(`${BASE_URL}${url}`, options);
+});
+
+Cypress.Commands.add(
+  'login',
+  (username = Cypress.env('username'), password = Cypress.env('password')) => {
+    cy.visit('/admin');
+    cy.get('#username').type(username);
+    cy.get('#login').type(password);
+    cy.get('.action-login').click();
+    cy.get('.page-title').should('have.text', 'Dashboard');
```

Comment:
A bit silly, but wouldn't `cy.contains('.page-title', 'Dashboard')` suffice? It's just to maintain consistency, as you are using several of these `contains(selector, text)` in other parts of the tests.